libs/velo-external-db-core/src/converters/filter_transformer.spec.ts

```diff
@@ -104,6 +104,18 @@ describe('Filter Transformer', () => {
                 })
             })
         })
+
+    })
+
+    describe('handle filter by date', () => {
+        test('transform velo date to date object', () => {
+            const filter = {
+                [ctx.fieldName]: { $gt: genCommon.veloDate() }
```

Comment:
can you create new 'data' property in CTX and use it here insted of generating new dates inside a test?